doc/pydoc/SGMM.md

```diff
@@ -0,0 +1,395 @@
+```
+
+**Returns**
+```
+self : SGMM
+	Itself, now updated with fitted parameters.
+```
+
+## _fit_stabilize
+```python
+SGMM._fit_stabilize(data)
+```
+A single attempt to estimate model parameters
+with the EM algorithm.
+
+The method repeatedly converges for various n_cores
+to pinpoint optimal n_cores. It does so by determining
+a search interval that contains the optimal n_cores and
+repeatedly narrows the interval until the optimal n_cores
+is determined.
+
+**Parameters**
+```
+data : array-like, shape (n_samples, n_features)
+	List of `n_features`-dimensional data points.
+	Each row corresponds to a single data point.
+```
+
+**Returns**
+```
+inertia : float
+	Log likelihood of the model.
+
+cores : array-like, shape (n_cores,)
+	A list of Cores for this fit trial.
+```
+
+## _truncate_interval
+```python
+SGMM._truncate_interval(data, interval, abic)
+```
+Truncate the interval by reducing the upper or lower limit
+based on which has the higher BIC/AIC (ABIC).
+
+**Parameters**
+```
+data : array-like, shape (n_samples, n_features)
+	List of `n_features`-dimensional data points.
+	Each row corresponds to a single data point.
+
+interval : tuple, shape (2,)
+	The interval which contains the optimal number of Cores.
+	Interpreted as [min, max].
+
+abic : tuple, shape (2,)
+	The abic scores corresponding to the interval.
+```
+
+**Returns**
+```
+interval : tuple, shape (2,)
+	The interval which contains the optimal number of Cores.
+	Interpreted as [min, max).
+
+abic : tuple, shape (2,)
+	The abic scores corresponding to the interval.
+```
+
+## _halve_interval
+```python
+SGMM._halve_interval(data, interval, abic, midpoint, abic_m)
+```
+Halve the interval based on the BIC/AIC (ABIC) of the midpoint.
+
+**Parameters**
+```
+data : array-like, shape (n_samples, n_features)
+	List of `n_features`-dimensional data points.
+	Each row corresponds to a single data point.
+
+interval : tuple, shape (2,)
+	The interval which contains the optimal number of Cores.
+	Interpreted as [min, max].
+
+abic : tuple, shape (2,)
+	The abic scores corresponding to the interval.
+
+midpoint : int
+	The midpoint of the interval.
+
+abic_m : float
+	The abic score corresponding to the midpoint.
+```
+
+**Returns**
+```
+interval : tuple, shape (2,)
+	The interval which contains the optimal number of Cores.
+	Interpreted as [min, max].
+
+abic : tuple, shape (2,)
+	The abic scores corresponding to the interval.
+```
+
+## _orient_stabilizer
+```python
+SGMM._orient_stabilizer(data)
+```
+Orient the initial interval for the stabilizer.
+
+**Parameters**
+```
+data : array-like, shape (n_samples, n_features)
+	List of `n_features`-dimensional data points.
+	Each row corresponds to a single data point.
+```
+
+**Returns**
+```
+interval : tuple, shape (2,)
+	The interval which contains the optimal number of Cores.
+	Interpreted as [min, max).
+
+bic : tuple, shape (2,)
+	The bic scores corresponding to the interval.
+```
+
+## _estimate_parameters
```

Comment:
Remove the following as they belong in [GMM.md](https://github.com/paradoxysm/dsigm/tree/master/doc/pydoc/GMM.md)